src/Module.cpp

```diff
@@ -756,9 +756,10 @@ void compile_multitarget(const std::string &fn_name,
             user_error << "All Targets must have matching arch-bits-os for compile_multitarget.\n";
         }
         // Some features must match across all targets.
-        static const std::array<Target::Feature, 8> must_match_features = {{
+        static const std::array<Target::Feature, 9> must_match_features = {{
```

Comment:
Should we keep this in sync with Target::get_runtime_compatible_target?

Reply:
https://github.com/halide/Halide/blob/212b9e5ec33b8563aa7ef878fffa93cdabdb99f7/src/Target.cpp#L889